tempesta_fw/t/functional/tests/helpers/be.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+__author__ = 'Tempesta Technologies Inc.'
+__copyright__ = 'Copyright (C) 2016 Tempesta Technologies Inc. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+# A primitive back-end HTTP server implementation suitable for testing purposes.
+
+import os
+import sys
+import signal
+import SimpleHTTPServer
+import BaseHTTPServer
+from  SimpleHTTPServer import SimpleHTTPRequestHandler
+import socket
+import datetime
+import time
+import setproctitle
```

Comment:
We need to install the module since it's not installed by default. So the requirement must be added to README. Do you really need the module?